controller/engine_controller.go

```diff
@@ -1768,30 +1781,35 @@ func (ec *EngineController) startRebuilding(e *longhorn.Engine, replica, addr st
 			// it is past the Backoff period, we'll try to mark the Replica as Failed and increase the Backoff period
 			// for the next failure.
 			if !ec.backoff.IsInBackOffSinceUpdate(e.Name, time.Now()) {
-				rep, err := ec.ds.GetReplica(replica)
+				if err := ec.updateReplicaRebuildFailedCondition(replica, replicaRebuildErrMsg); err != nil {
+					log.WithError(err).Errorf("Failed to update rebuild status information on replica %v", replicaName)
+					return
+				}
```

Comment:
Change to below. The same reason as above. 

```go
replica, err = ec.updateReplicaRebuildFailedCondition(replica, replicaRebuildErrMsg)
if err != nil {
    log.WithError(err).Errorf("Failed to update rebuild status information on replica %v", replicaName)
    return
}
```

Reply:
Changed. Thanks